features/cis_os/exec.config

```diff
@@ -59,7 +59,8 @@ echo "auth	required	pam_wheel.so" >> /etc/pam.d/su
 
 # 99.1.1.23: Disable USB Dev
 echo 'ACTION=="add", SUBSYSTEMS=="usb", TEST=="authorized_default", ATTR{authorized_default}="0"' >> /etc/udev/rules.d/10-CIS_99.2_usb_devices.sh
-echo 'ACTION=="add", ATTR{bDeviceClass}=="09", TEST=="authorized", ATTR{authorized}="1"' >> /etc/udev/rules.d/10-CIS_99.2_usb_devices.shecho 'ACTION=="add", ATTR{product}=="*[Kk]eyboard*", TEST=="authorized", ATTR{authorized}="1"' >> /etc/udev/rules.d/10-CIS_99.2_usb_devices.sh
+echo 'ACTION=="add", ATTR{bDeviceClass}=="09", TEST=="authorized", ATTR{authorized}="1"' >> /etc/udev/rules.d/10-CIS_99.2_usb_devices.sh
```

Comment:
Not quite sure if this belongs to this PR. However, it seems to fix the newline issue (`evices.shecho`).